tests/integration/targets/v3.5/tasks/netbox_vm_interface.yml

```diff
@@ -54,14 +54,14 @@
       mac_address: "00:00:00:AA:AA:01"
       description: "Updated test100-vm"
       mode: Tagged
-      untagged_vlan:
-        name: Wireless
-        site: Test Site
-      tagged_vlans:
-        - name: Data
-          site: Test Site
-        - name: VoIP
-          site: Test Site
+      #untagged_vlan:
+      #  name: Wireless
+      #  site: Test Site
+      #tagged_vlans:
+      #  - name: Data
+      #    site: Test Site
+      #  - name: VoIP
+      #    site: Test Site
```

Comment:
These are commented out, should they be removed? Or did the API change and that's why these are commented out?